knowledge_base/Component-Kit.md

```diff
@@ -4,6 +4,22 @@ The component kit is Engineering’s current implementation of that source of tr
 
 A rendered version of our component kit can be [viewed on the live site](https://commonwealth.im/components), or found in the codebase in the `component_kit` directory.
 
+Components page got revamped and this also means that we will structure the components in a bit more readable way. There are two groups of building blocks in our kit:
```

Comment:
Possible to add a specific datestamp here? E.g. "As of 240109, the components page has been revamped"

Reply:
done